physics/physpkg.F90

```diff
@@ -1357,7 +1382,11 @@ subroutine tphysac (ztodt,   cam_in,  &
     use waccmx_phys_intr,   only: waccmx_phys_ion_elec_temp_tend ! WACCM-X
     use aoa_tracers,        only: aoa_tracers_timestep_tend
     use physconst,          only: rhoh2o
+#ifdef OSLO_AERO
+    use oslo_aero_model,    only: aero_model_drydep
```

Comment:
Why isn't `oslo_aero_model` just a shadow file called `aero_model`?

Reply:
Because its totally different. And I wanted to have all the oslo_aero code to have a prefix of oslo_aero to eventually think of putting this in another model rather than having a name which matches cam.